Voycar.Api.Web/Generic/Endpoint/Post/Single.cs

```diff
@@ -0,0 +1,35 @@
+namespace Voycar.Api.Web.Generic.Endpoint.Post;
+
+using Repository;
+
+public abstract class Single<TEntity>
+    : Endpoint<TEntity, Results<Ok, NoContent>>
+    where TEntity : Entity
+{
+    private readonly IRepository<TEntity> _repository;
+    private readonly string[] roles;
+
+    protected Single(IRepository<TEntity> repository, string[] roles)
+    {
+        this._repository = repository;
+        this.roles = roles;
+    }
+
+    public override void Configure()
+    {
+        this.Post(typeof(TEntity).Name.ToLowerInvariant());
+        this.Roles(this.roles);
+    }
+
+    public override async Task HandleAsync(TEntity req, CancellationToken ct)
+    {
+        var created = this._repository.Create(req);
+
+        if (created)
+        {
+            await this.SendResultAsync(TypedResults.Ok());
+            return;
+        }
+        await this.SendResultAsync(TypedResults.NoContent());
```

Comment:
```suggestion
        }
        
        await this.SendResultAsync(TypedResults.NoContent());
```
in den anderen Klassen ist da noch n Line Break